client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions-i2/pending-actions/style.scss

```diff
@@ -22,7 +22,16 @@
 
 	.pending-actions__action--success {
 		.pending-actions__action-icon {
-			fill: #008a20;
+			svg {
+				fill: #008a20;
```

Comment:
Nit, we should be using color variables rather than hard-coded HEX values.  This one is `var( '--studio-green-50' )` or `var( --color-success )`